QualityControl/public/frameworkInfo/FrameworkInfo.js

```diff
@@ -31,19 +31,28 @@ export default class FrameworkInfo extends Observable {
 
   /**
    * Load FrameworkInfo into `item`
+   * @param {Components} components Array of components from which retrieve information
    * @returns {undefined}
    */
-  async getFrameworkInfo() {
+  async getFrameworkInfo(components) {
     this.item = RemoteData.loading();
     this.notify();
 
-    const { result, ok } = await this.model.loader.get('/api/status/framework');
-    if (!ok) {
-      this.item = RemoteData.failure(result.message);
+    try {
+      const results = {};
+      await Promise.all(components.map(async (component) => {
```

Comment:
Do I understand correctly that these are just migration changes and the changes for front-end will come in a future PR?
Because in these changes, the status cards will still be loaded all at one time compared to the requirement of the ticket to load independently